delivery_carrier_package_measure_required/models/delivery_carrier.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = "delivery.carrier"
+
+    package_height_required = fields.Boolean(string="Height required")
```

Comment:
A carrier can transport different delivery packages requiring different measurement.
I would move this from `delivery.carrier` to `product.packaging` (delivery package)

Reply:
I moved the fields to 'product.packaging' but did not rename the module